### PR TITLE
Add cookie login flow

### LINKED ISSUE #6: Add ability to specify IP address to test BMC locally
## Overview
This is partially working using an environmental variable. We are not able to persist our authenticated state with our temporary workflow. We need a permanent solution that works with both dev and production

### PR DESCRIPTION
**Pull request by yoshiemuranaka**
Add cookie based authentication

Closes #6

### CODE REVIEW

**Comment by yoshiemuranaka** [comment]
Not sure if this works for production/overlay. Can test with overlay when #38 is merged.

File: src/store/modules/Authentication/AuthenticanStore.js [hunk 1]
```diff
@@ -3,52 +3,39 @@ import api from "../../api";
...
   state: {
-    auth: {},
-    status: "",
-    token: sessionStorage.getItem("token") || ""
+    status: ""
   },
   getters: {
     authStatus: state => state.status,
-    isLoggedIn: state => !!state.token
+    isLoggedIn: (_, getters) => getters.authStatus === "authenicated"
```
Comment (gtmills) [inline src/store/modules/Authentication/AuthenticanStore.js]: spelling ? authenticated

Comment (yoshiemuranaka) [reply src/store/modules/Authentication/AuthenticanStore.js]: Thanks

**Review by dixsie — APPROVED** [review]

**Review by derick-montague — CHANGES_REQUESTED** [review]
We found some issues on production that we are trying to resolve.

**Comment by derick-montague** [comment]
I noticed that refreshing the page will redirect browser to login. Refreshing the page on login throws an error.

File: babel.config.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ["@vue/cli-plugin-babel/preset"]
+  presets: [["@babel/preset-env", { useBuiltIns: "entry", corejs: 3 }]]
```
Comment (derick-montague) [inline babel.config.js]: We only need to change the useBuiltIns value. By default, vue cli-plugin-babel/preset sets the useBuiltIns to  `usage`. If we just update it to use `entry`, we are good. 

`  presets: [["@vue/cli-plugin-babel/preset", { useBuiltIns: "entry" }]]`

Comment (yoshiemuranaka) [reply babel.config.js]: Cool, amended my last commit

**Review by derick-montague — APPROVED** [review]
Works great! Thank you, @yoshiemuranaka

**Comment by derick-montague** [comment]
Integrated into webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28638

**Pull request merged**